### PR TITLE
docs: update scripts/readme.md

### PR DESCRIPTION
**Pull request by danieljhegeman**
## Reason for Change

- The `scripts/readme.md` was updated recently as part of the introduction of `cellxgene-ontology-guide`
- Some instructions needed to be refined in the wake of these updates
- Old/incomplete instructions needed to be removed to prevent confusion

## Changes

- [See readme.md]

## Testing

- Either list QA steps or reasoning you feel QA is unnecessary
- Reminder For CLI changes: upon merge, contact Lattice for final sign-off. Do not release a new cellxgene-schema 
version to PyPI without explicit QA + sign-off from Lattice on all functional CLI changes. They may install the package
version at HEAD of main with 
```
pip install git+https://github.com/chanzuckerberg/single-cell-curation/@main#subdirectory=cellxgene_schema_cli
```

## Notes for Reviewer

### CODE REVIEW

File: cellxgene_schema_cli/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-.PHONY: update-references
-update-references: gene-processing clean
+.PHONY: update-gene-references
```
Comment (danieljhegeman) [inline cellxgene_schema_cli/Makefile:1]: this is no longer a generic operation for ontology and genes alike. It is now just genes and should be labeled as such.

File: cellxgene_schema_cli/scripts/readme.md [hunk 2]
```diff
@@ -35,28 +34,18 @@ The following gene references are currently stored (to see specific versions ref
...
-To download all these files the following has to be called:
-
-If reference gene versions need to be updated then `Makefile` has to be changed accordingly under the rules corresponding to each organism.
-
-## Cleanup
-
-If instead of the rule `update-references` individual make rules were executed, then make sure to remove all unused downloads using:
-```bash
-make clean
-```
```
Comment (danieljhegeman) [inline cellxgene_schema_cli/scripts/readme.md:62]: I judge this to all be outdated/irrelevant. We no longer have make recipes for individual files.

**Review by Bento007 — APPROVED** [review]

File: cellxgene_schema_cli/scripts/readme.md [hunk 3]
```diff
@@ -1,25 +1,24 @@
 # Updating reference files
 
 ## Quick start 
-To update the gene and ontology references the following files need to be modified with the updated references:
+To update the gene references, update `cellxgene_schema_cli/cellxgene_schema/gencode_files/gene_info.yml` with new references.
 
-- `cellxgene_schema_cli/cellxgene_schema/gencode_files/gene_info.yml`
-- Bump cellxgene-ontology-guide version to latest release with updated ontologies in `cellxgene_schema_cli/cellxgene_schema/requirements.txt`
+To update ontology references, update the `cellxgene-ontology-guide` version in `cellxgene_schema_cli/requirements.txt`.
```
Comment (danieljhegeman) [inline cellxgene_schema_cli/scripts/readme.md:6]: The directory for the requirements file was wrong

File: cellxgene_schema_cli/scripts/readme.md [hunk 4]
```diff
@@ -1,25 +1,24 @@
...
 
 ## Updating ontologies
 
-Ontology information for cellxgene-schema is derived from cellxgene-ontology-guide. To update the ontologies in 
+Ontology information for cellxgene-schema is derived from `cellxgene-ontology-guide`. To update the ontologies in 
 conjunction with a schema release, update [ontology_info.json in cellxgene-ontology-guide](https://github.com/chanzuckerberg/cellxgene-ontology-guide/blob/main/ontology-assets/ontology_info.json),
-and then generate a new release with the subsequently generated ontology reference files. Bump the version of cellxgene-ontology-guide in 
-`cellxgene_schema_cli/cellxgene_schema/requirements.txt` to this latest release.
+and then generate a new release with the subsequently generated ontology reference files. Bump the version of `cellxgene-ontology-guide` in 
+`cellxgene_schema_cli/requirements.txt` to this latest release.
```
Comment (danieljhegeman) [inline cellxgene_schema_cli/scripts/readme.md:21]: same issue with directory

**Review by nayib-jose-gloria — APPROVED** [review]

**Pull request merged**